Quaver throws an error on the console when querying an age restricted track or a query that has no results

**Describe the bug**
What isn't working as intended, and what does it affect?
Affects functionality

**Severity**
- [x] Critical
- [ ] High
- [ ] Medium
- [ ] Low

**Affected versions**
What versions are affected by this bug? (e.g. >=3.0.1, 2.5.1-2.6.3, >=1.2.0)
4.0.0-next.*

**Steps to reproduce**
Steps to reproduce the behavior. (e.g. click on a button, enter a value, etc. and see error)

1. Query a track with Quaver:

For the loadType called `NO_RESULTS`
```
Bittersweet Sangatsu no Phantasia mapped by Deca
```

For the loadType called `LOAD_FAILED`
```
https://www.youtube.com/watch?v=x8VYWazR5mE
```

**Expected behavior**
What is expected to happen?
When one of them are invoked they should `return;` instead of `break;` so the queueing procedure will not continue because they're supposed to be errors.

**Actual behavior**
What actually happens? Attach or add errors or screenshots here as well.

`NO_RESULTS` and `LOAD_FAILED` loadTypes do not return when invoked causing unintended behavior

See error:

For the loadType called `NO_RESULTS`
```js
$ node .
2022-07-12T15:01:47.484Z [Discord] INFO: Connected. Logged in as Aia#3990.
2022-07-12T15:01:47.492Z [Quaver] INFO: Running version 4.0.0. For help, see https://github.com/ZPTXDev/Quaver/issues.
2022-07-12T15:01:49.531Z [Lavalink] INFO: Connected.
2022-07-12T15:02:16.814Z [Quaver] INFO: [G 906471497231630336 | U 250934589621665793] Processing command play
2022-07-12T15:02:16.815Z [Quaver] INFO: [G 906471497231630336 | U 250934589621665793] Executing command play
2022-07-12T15:02:20.059Z [Quaver] ERROR: [G 906471497231630336 | U 250934589621665793] Encountered error with command play
2022-07-12T15:02:20.061Z [Quaver] ERROR: Cannot find module './locales/en/.json'
Require stack:
- C:\Users\Ava\Documents\GitHub\Quaver\functions.js
- C:\Users\Ava\Documents\GitHub\Quaver\main.js
Error: Cannot find module './locales/en/.json'
Require stack:
- C:\Users\Ava\Documents\GitHub\Quaver\functions.js
- C:\Users\Ava\Documents\GitHub\Quaver\main.js
    at Module._resolveFilename (node:internal/modules/cjs/loader:946:15)
    at Module._load (node:internal/modules/cjs/loader:787:27)
    at Module.require (node:internal/modules/cjs/loader:1012:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at getLocale (C:\Users\Ava\Documents\GitHub\Quaver\functions.js:137:16)
    at ReplyHandler.locale (C:\Users\Ava\Documents\GitHub\Quaver\classes\ReplyHandler.js:98:27)
    at async Object.execute (C:\Users\Ava\Documents\GitHub\Quaver\commands\play.js:129:3)     
    at async Object.execute (C:\Users\Ava\Documents\GitHub\Quaver\events\interactionCreate.js:85:5)
```

For the loadType called `LOAD_FAILED`
```js
$ node .
2022-07-12T15:04:08.306Z [Discord] INFO: Connected. Logged in as Aia#3990.
2022-07-12T15:04:08.311Z [Quaver] INFO: Running version 4.0.0. For help, see https://github.com/ZPTXDev/Quaver/issues.
2022-07-12T15:04:08.615Z [Quaver] INFO: [G 906471497231630336 | U 250934589621665793] Processing command play
2022-07-12T15:04:08.619Z [Quaver] INFO: [G 906471497231630336 | U 250934589621665793] Executing command play
2022-07-12T15:04:10.322Z [Lavalink] INFO: Connected.
2022-07-12T15:04:12.119Z [Quaver] ERROR: [G 906471497231630336 | U 250934589621665793] Encountered error with command play
2022-07-12T15:04:12.122Z [Quaver] ERROR: Cannot find module './locales/en/.json'
Require stack:
- C:\Users\Ava\Documents\GitHub\Quaver\functions.js
- C:\Users\Ava\Documents\GitHub\Quaver\main.js
Error: Cannot find module './locales/en/.json'
Require stack:
- C:\Users\Ava\Documents\GitHub\Quaver\functions.js
- C:\Users\Ava\Documents\GitHub\Quaver\main.js
    at Module._resolveFilename (node:internal/modules/cjs/loader:946:15)
    at Module._load (node:internal/modules/cjs/loader:787:27)
    at Module.require (node:internal/modules/cjs/loader:1012:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at getLocale (C:\Users\Ava\Documents\GitHub\Quaver\functions.js:137:16)
    at ReplyHandler.locale (C:\Users\Ava\Documents\GitHub\Quaver\classes\ReplyHandler.js:98:27)
    at async Object.execute (C:\Users\Ava\Documents\GitHub\Quaver\commands\play.js:129:3)     
    at async Object.execute (C:\Users\Ava\Documents\GitHub\Quaver\events\interactionCreate.js:85:5)
```

--- a/commands/play.js
+++ b/commands/play.js
@@ -89,10 +89,10 @@ module.exports = {
 				}
 				case 'NO_MATCHES':
 					await interaction.replyHandler.localeError('CMD_PLAY_NO_RESULTS');
-					break;
+					return;
 				case 'LOAD_FAILED':
 					await interaction.replyHandler.localeError('CMD_PLAY_LOAD_FAILED');
-					break;
+					return;
 				default:
 					await interaction.replyHandler.localeError('DISCORD_CMD_ERROR');
 					return;

